Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOneV3/TrendMicroVisionOneV3.yml

```
@@ -378,6 +378,15 @@ script:
       name: top
     - description: 'List of fields to include in the search results. If no fields are specified, the query returns all supported fields. e.g. select="dpt,dst,endpointHostName".'
       name: select
+    - description: 'Do you want to fetch all matching records or only records matching the top value.'
+      name: fetch_all
+      auto: PREDEFINED
+      predefined:
+      - "true"
+      - "false"
+      default: false
+    - description: 'Max results to be fetched by call.'
+      name: fetch_max_count
```

Lets add the default value here, rather than having a magic number in the code (5000)
```suggestion
      name: fetch_max_count
      defaultvalue: 5000
```

---

done.